.github/issue_templates/docs-bug.yaml

```diff
@@ -0,0 +1,35 @@
+name: Docs Bug 🐞 report
+description: Report mistakes and/or bugs in LocalStack Docs.
+title: "[Docs Bug 🐞 report]: "
+labels: "🐞 docs bug"
+assignees: quetzalliwrites, HarshCasper
```

Comment:
Should we be the automatic assignees, Harsh? It seems logical, just not sure if anyone else wants to be added. 😼